src/views/SystemProfile/index.test.tsx

```diff
@@ -30,7 +30,6 @@ describe('The System Profile view', () => {
           history={mockHistory}
           location={mockLocation}
           auth={null}
-          searchResults={[]}
```

Comment:
What is the point of removing this, and can we remove auth?

Reply:
We removed searchResults because it's not actually a prop to the component. I'm not sure why it was there

Reply:
And yes, Auth can be removed! Good catch.